types/smb2/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for smb2 0.2
+// Project: https://github.com/bchelli/node-smb2
+// Definitions by: Nick Whitlock <https://github.com/eezstreet>
+// Definitions: https://github.com/eezstreet/DefinitelyTyped
+
+export = SMB2;
+
+interface SMB2Options {
+	share: string;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/smb2/index.d.ts;
  Line: 9;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.